NEWS

```diff
@@ -147,6 +147,10 @@ CHANGES WITH 246 in spe:
         * networkd.conf gained a new boolean setting ManageForeignRoutes=. If
           enabled systemd-networkd manages all routes configured by other tools.
 
+        * .network files managed by systemd-networkd gained a new section
+          [SR-IOV], in order to configure various properties of interface's
+          virtual functions.
```

Comment:
We should write "in order to configure  SR-IOV capable network devices .

Reply:
Thank you for the suggestion.